CDC-Data-Reconciliation-Backend/server.py

```diff
@@ -62,6 +62,19 @@ async def manual_report(state_file: UploadFile = File(None), cdc_file:  UploadFi
 
     return res
 
+# Runs the query on the state NBS ODSE database and outputs the results to a CSV file
+def automatic_report(year: int):
+    state_results = run_query(year)
+
+    folder_name = "temp"
+    if not os.path.exists(folder_name):
+        os.makedirs(folder_name)
+    result_save_to = f"./{folder_name}/state_results.csv"
+    with open(result_save_to, "w", newline='') as f:
+        csv_out = csv.writer(f)
+        #csv_out.writerow(["column", "titles", "needed?"])
+        for row in state_results:
+            csv_out.writerow(row)
```

Comment:
There should be more to this endpoint. You are missing accepting the CDC file from the frontend and running the comparison and returning the results of the comparison back to the client. You currently are just saving the state data to a csv file.